tools/example-module/generate-example-module.ts

```diff
@@ -27,20 +27,25 @@ function inlineExampleModuleTemplate(parsedData: ExampleMetadata[]): string {
     return result.concat(data.component).concat(data.additionalComponents);
   }, [] as string[]).join(',');
 
-  const exampleComponents = parsedData.reduce((result, data) => {
-    result[data.id] = {
-      title: data.title,
-      component: data.component,
-      additionalFiles: data.additionalFiles,
-      selectorName: data.selectorName.join(', '),
-    };
-
-    return result;
+  const exampleComponents = parsedData.map((metaData) => {
```

Comment:
Should these changes be reverted? They're sorted out in https://github.com/angular/material2/pull/13955.